src/components/Graph/Graph.js

```diff
@@ -0,0 +1,65 @@
+import React from 'react';
+import useStore from '../../services/useStore';
+import { Chart, LineElement, LinearScale, CategoryScale, BarElement, PointElement } from 'chart.js';
+Chart.register(LineElement, LinearScale, CategoryScale, BarElement, PointElement);
```

Comment:
I'd place this further below since it's not an import like the lines following.

Reply:
Thanks and done!